src/content/docs/fr/reference/errors/session-config-missing-error.mdx

```diff
@@ -10,4 +10,4 @@ githubURL: https://github.com/withastro/astro/blob/main/packages/astro/src/core/
 Émis lorsque le stockage de session est activé mais non configuré.
 
 **Voir aussi :**
--  [experimental.session](/fr/reference/experimental-flags/sessions/)
+-  [Le guide des sessions](/fr/guides/sessions/)
```

Comment:
The French translations opt not to translate API names like this, so we can just keep it the same.
```suggestion
- [Sessions](/fr/guides/sessions/)
```